src/tables.css

```diff
@@ -31,7 +31,7 @@
  *
  */
 .table,
-.prose table {
+.prose table:not(.unprose) {
```

Comment:
I'm good with this (apply `unprose` directly to elements), as long as there's a little more explanation in the docs. However, if you really favor the container, I'm ok with you making the choice.